### ISSUE TITLE
Words that exceed the width sometimes disappear when "text-overflow: ellipsis" is applied.

### ISSUE THREAD

**Issue by yukihiroK** [vercel/satori#394]
# Bug report

## Description / Observed Behavior
I am not sure if this is due to Satori or the browser, a word that exceeds the width sometimes disappears instead of ending with "..." when "text-overflow: ellipsis" is applied.

As can be seen in the second line of the Playground, simply placing a symbol such as a period or comma after a word that exceeds the width of the word will cause the word to no longer be rendered. Similarly, a word (sequences of characters not separated by spaces) in CJK characters seems to be disappeared if it is too long.
I am not quite sure how this happens - in some cases, such as the fourth line of the example, it seems to work properly, perhaps due to the length of the word.

## Expected Behavior
Should text end with "..." if its length exceeds the width.

## Reproduction
![81a73bcb-6705-45ab-88b3-1f884418824d](https://user-images.githubusercontent.com/57009247/218047408-39dff971-1d1a-4a6f-8033-56571a7b05d7.png)

[Playground](https://og-playground.vercel.app/?share=vVRNb9NAEP0ro5VQpCo0KWo5RAWEAkiUA0ip1IsvG3scb7retXbXdU3kQ8IJxJ07RxAXzkj8GAt-B7P-SJtwQgJ88Meb8cx7-3Z2xUIdIZuw00hcBQrAulLig9XKvwNEwmaSlxMYxBKvB8MWTVAsEkfg0Xh8pwcLEblkD-NSLNRzh6mlQIjKoelDy9w6EZdTTaDytXbDvt0TYTB0Qisf1TJPVR-d8_ByYXSuoqmW2lC8SITD7c9UdCZe4wTuj29BFx3t43GLVlWgHgbKK9-RvRUdME8jYE32HqOAFs5T6qKd-uOTtrS-QhNLXRCzREQRdswblrOMh0RtoHRheOYDVUU0AHoL9k34rbq_HF67lzddUEqRWWH7JfDaAJqyAKuAnSfCQqHNpYVQG69ClgFrkk5H1PcfExgdwDNtwOoUwSC3Wg1BxMAhQyN0NCRSacqBUrRLkBLLdK4lEGdyIkRKINjzj7w7PMuQGwtCwYw7bcQhHIyafo3WQrhE525b_fAPhEquFt7cJb979qrz9i8twGOwIhWSmxsJXIUIFmk-wGnQYZgb6uQSmJ69gDChhJBmwu7Km7VVZEmb2kC9_lpv3tbr9_X60_8y1KvhKqLtXJIv240VC4WPdqheNPjPbx9_fPlQbz7Xm-_-_ubdHtHb7_2TDZnO_KhZNlmRpUSXTU6O7g1Ze_p0HxHO8wWbOJPjkGGql-K8zPx5FsucDhSq4if_aTrHqM2qql8)

## Additional Context
I found this bug as when I tried to apply ellipsis to a Japanese sentence, the whole sentence disappeared.
Running Playground in Firefox seems to give the intended result.

**Comment by shuding** [comment]
@LuciNyan I think your PR fixed the problem, but another issue appears that the last two lines are gone:

- After #372: [playground](https://og-playground.vercel.app/?share=vVRNb9NAEP0ro5VQpCo0KWo5RAWEAkiUA0ip1IsvG3scb7retXbXdU3kQ8IJxJ07RxAXzkj8GAt-B7P-SJtwQgJ88Meb8cx7-3Z2xUIdIZuw00hcBQrAulLig9XKvwNEwmaSlxMYxBKvB8MWTVAsEkfg0Xh8pwcLEblkD-NSLNRzh6mlQIjKoelDy9w6EZdTTaDytXbDvt0TYTB0Qisf1TJPVR-d8_ByYXSuoqmW2lC8SITD7c9UdCZe4wTuj29BFx3t43GLVlWgHgbKK9-RvRUdME8jYE32HqOAFs5T6qKd-uOTtrS-QhNLXRCzREQRdswblrOMh0RtoHRheOYDVUU0AHoL9k34rbq_HF67lzddUEqRWWH7JfDaAJqyAKuAnSfCQqHNpYVQG69ClgFrkk5H1PcfExgdwDNtwOoUwSC3Wg1BxMAhQyN0NCRSacqBUrRLkBLLdK4lEGdyIkRKINjzj7w7PMuQGwtCwYw7bcQhHIyafo3WQrhE525b_fAPhEquFt7cJb979qrz9i8twGOwIhWSmxsJXIUIFmk-wGnQYZgb6uQSmJ69gDChhJBmwu7Km7VVZEmb2kC9_lpv3tbr9_X60_8y1KvhKqLtXJIv240VC4WPdqheNPjPbx9_fPlQbz7Xm-_-_ubdHtHb7_2TDZnO_KhZNlmRpUSXTU6O7g1Ze_p0HxHO8wWbOJPjkGGql-K8zPx5FsucDhSq4if_aTrHqM2qql8)
- Before #372: [playground](https://satori-playground-4iuygs1n0.vercel.sh/?share=vVRNb9NAEP0ro5VQpCo0KWo5RAWEAkiUA0ip1IsvG3scb7retXbXdU3kQ8IJxJ07RxAXzkj8GAt-B7P-SJtwQgJ88Meb8cx7-3Z2xUIdIZuw00hcBQrAulLig9XKvwNEwmaSlxMYxBKvB8MWTVAsEkfg0Xh8pwcLEblkD-NSLNRzh6mlQIjKoelDy9w6EZdTTaDytXbDvt0TYTB0Qisf1TJPVR-d8_ByYXSuoqmW2lC8SITD7c9UdCZe4wTuj29BFx3t43GLVlWgHgbKK9-RvRUdME8jYE32HqOAFs5T6qKd-uOTtrS-QhNLXRCzREQRdswblrOMh0RtoHRheOYDVUU0AHoL9k34rbq_HF67lzddUEqRWWH7JfDaAJqyAKuAnSfCQqHNpYVQG69ClgFrkk5H1PcfExgdwDNtwOoUwSC3Wg1BxMAhQyN0NCRSacqBUrRLkBLLdK4lEGdyIkRKINjzj7w7PMuQGwtCwYw7bcQhHIyafo3WQrhE525b_fAPhEquFt7cJb979qrz9i8twGOwIhWSmxsJXIUIFmk-wGnQYZgb6uQSmJ69gDChhJBmwu7Km7VVZEmb2kC9_lpv3tbr9_X60_8y1KvhKqLtXJIv240VC4WPdqheNPjPbx9_fPlQbz7Xm-_-_ubdHtHb7_2TDZnO_KhZNlmRpUSXTU6O7g1Ze_p0HxHO8wWbOJPjkGGql-K8zPx5FsucDhSq4if_aTrHqM2qql8)
